csync/src/csync_statedb.c

```diff
@@ -226,6 +226,8 @@ int csync_statedb_close(CSYNC *ctx) {
   return rc;
 }
 
+#define METADATA_COLUMNS "phash, pathlen, path, inode, uid, gid, mode, modtime, type, md5, fileid, remotePerm, filesize, ignoredChildrenRemote, contentChecksum, contentChecksumTypeId"
```

Comment:
we could save some CPU cycle by not reading stuff lie uid, gid, mode  (long terme we should remove them totaly)